ray-operator/apis/ray/v1alpha1/raycluster_types.go

```diff
@@ -112,6 +112,8 @@ type RayClusterStatus struct {
 	LastUpdateTime *metav1.Time `json:"lastUpdateTime,omitempty"`
 	// Service Endpoints
 	Endpoints map[string]string `json:"endpoints,omitempty"`
+	// Metadata
+	Metadata map[string]string `json:"metadata,omitempty"`
```

Comment:
@scarlet25151 originally wrote [here](https://github.com/davidxia/kuberay/pull/1#pullrequestreview-1045435014)

> Here I think it's not necessary for us to add some new spec for the crd, for the metadata we can just store then as some annotation in rayclusters object.

@Jeffwan, are you OK with the K8s annotation approach or prefer some other way?

Reply:
annotation sounds better. We try not to update CRD as much as we can